src/diffpy/morph/morphs/morphsqueeze.py

```diff
@@ -9,9 +9,29 @@ class MorphSqueeze(Morph):
 
     Configuration Variables
     -----------------------
-    squeeze
-        list or array-like
-        Polynomial coefficients [a0, a1, ..., an] for the squeeze function.
+    squeeze : list
+        The polynomial coefficients [a0, a1, ..., an] for the squeeze function
+        where the polynomial would be of the form a0 + a1*x + a2*x^2 and so
+        on.  The order of the polynomial is determined by the length of the
+        list.
+
+    Example
+    -------
+    >>> import numpy as np
+    >>> from numpy.polynomial import Polynomial
+    >>> from diffpy.morph.morphs.morphsqueeze import MorphSqueeze
+
+    >>> x_target = np.linspace(0, 10, 101)
+    >>> y_target = np.sin(x_target)
+    >>> x_morph = np.linspace(0, 10, 101)
+    >>> squeeze_coeff = [0.1, -0.01, 0.005]
+    >>> poly = Polynomial(squeeze_coeff)
+    >>> y_morph = np.sin(x_morph + poly(x_morph))
+
+    >>> morph = MorphSqueeze()
```

Comment:
this block is what we are after.  We could add a few words about what the variables mean.  Maybe also add a couple of lines that show other attributes that the user can access from the `morph` instance.  For example things like
`x_morph_in = morph.x_morph_in` etc.  No need to be exhaustive, just give a few examples to help users.

btw, this looks great in general.  I am not sure if we have a "group standard" for these code blocks.  I think that it is possible to write htese actually so that they can be run by some automated testing codes but we don't really do that.  I just feel that, as a user, if there is a few lines of code showing how to use this, it can be super helpful.